server/db/migrations/20220411191359-add_account_profile_table_and_refactor_wallet.js

```diff
@@ -0,0 +1,92 @@
+'use strict';
+
+require('dotenv').config({
+    path: require('path').resolve(process.cwd(), '.env-server')
+});
+
+const Sequelize = require("sequelize");
+const {Fio, Ecc} = require("@fioprotocol/fiojs");
+const {PrivateKey} = Ecc;
+
+const walletPrivateKey = process.env.WALLET_PRIVATE_KEY;
+
+const walletPublicKey = PrivateKey(walletPrivateKey).toPublic().toString('FIO');
+
+const defaultActor = Fio.accountHash(walletPublicKey);
+const defaultPermission = 'active';
+
+module.exports = {
+  up: async (QI, DT) => {
+    await QI.createTable('account_profile',
+        {
+          id: {
+            type: DT.INTEGER,
+            primaryKey: true,
+            autoIncrement: true,
+            allowNull: false
+          },
+          actor: {
+            type: DT.STRING,
+            allowNull: false,
+          },
+          permission: {
+             type: DT.STRING,
+             allowNull: false
+          },
+          profile_name: {
+            type: DT.STRING,
+            allowNull: true
+          },
+          is_default: {
+            type: DT.BOOLEAN,
+            allowNull: false,
+            defaultValue: false
+          },
+          created: {
```

Comment:
there are predefined props for it = `created_at`, could we use it?

Reply:
add created_at, deleted_at (paranoid, timestamps: true) into migration and model